apps/cms/lib/page/event.ex

```diff
@@ -73,14 +75,21 @@ defmodule CMS.Page.Event do
           path_alias: String.t() | nil,
           paragraphs: [Paragraph.t()],
           registration_link: Link.t() | nil,
-          livestream_link: Link.t() | nil
+          livestream_link: Link.t() | nil,
+          agenda_embed: Agenda.t() | nil
```

Comment:
Alongside the other possible renames, I'll add making this simply `agenda` - not sure the _embed adds info

Reply:
There is already an `agenda` property (the deprecated one where you just copy paste in a bunch of text), which we still need to support for old events that won't be getting the "new" event agenda makeover (using this new field). The "new" agenda is an event reference type field, aka an "embedded" type field since the CMS API is going to "expand" the referenced node ID here into a full struct-compatible JSON response as the field value.

So, I don't want to call it `agenda` because that duplicates an existing prop. I ok with renaming the existing agenda field to something like `agenda_old`, or I guess, making the new one `agenda_new`.

I don't want to call it `agenda_reference` because it's not a simple integer reference. As mentioned above, the CMS is expanding this for us to a fully embedded chunk. There is precedence for this in one other field.

I think having `_reference` as an indicator of a simple integer, and `_embed` or `_nested` as an indicator of what's happening here (a struct inside a prop inside a struct), would be a good pattern/refactor, if that's what you would like to do.

Sorry for the confusion. Drupal!

Reply:
> I think having _reference as an indicator of a simple integer, and _embed or _nested as an indicator of what's happening here (a struct inside a prop inside a struct), would be a good pattern/refactor, if that's what you would like to do.

I think that makes sense, generally speaking. Thanks for this context.

@amaisano I'll propose this -- how about we rename `CMS.Page.Agenda` to `CMS.Page.EventAgenda`? Then we avoid the name collision confusion with the old agenda, we can rename this property to `event_agenda`, and I think it's a bit clearer overall.